astronomer/providers/amazon/aws/example_dags/example_emr_eks_containers_job.py

```diff
@@ -146,7 +146,7 @@ def create_emr_virtual_cluster_func(task_instance: Any) -> None:
     # [START howto_sensor_emr_job_container_async]
     emr_job_container_sensor = EmrContainerSensorAsync(
         task_id="emr_job_container_sensor",
-        job_id=run_emr_container_job.output,
+        job_id=str(run_emr_container_job.output),
```

Comment:
how / why was this working before and not now?

Reply:
this still works but` EmrContainerSensorAsync` has attribute `job_id`  which accepts type `str` but the output type is `XComArg` now so mypy complains  `Argument "job_flow_id" to "EmrTerminateJobFlowOperator" has incompatible type "XComArg"; expected "str"  [arg-type]`.

Reply:
Apply PEP-563 (Postponed Evaluation of Annotations) to core airflow (https://github.com/apache/airflow/pull/26290
https://github.com/apache/airflow/pull/26290)

Reply:
changes looking good to me